src/utilities/renavam/index.ts

```diff
@@ -0,0 +1,43 @@
+import { onlyNumbers } from '../../helpers';
+
+export const MIN_LENGTH = 9;
+export const LENGTH = 11;
+export const SUM = 284;
+
+export const isValid = (input: string) => {
+  if (!input || typeof input !== 'string' || input.length < MIN_LENGTH) return false;
+
+  let result = onlyNumbers(input.padStart(11, '0'));
+
+  if (!result.match('[0-9]{11}')) return false;
+
+  let resultWithoutDigit = result.substring(0, 10);
```

Comment:
You could do something like this: `result.substring(0, 10).split('').reverse().join('')` and use `const` insteade of `let`